### PR TITLE
 testing fork and merge request

### PR DESCRIPTION
**Pull request by youngerson**
testing fork and merge request

### CODE REVIEW

File: test/nodesetexporter/NodesetExporterLoopTest.cpp [hunk 1]
```diff
@@ -9,6 +9,8 @@
 // todo Redesign the tests into a more understandable form. Remove the generators, form everything manually so that you can see what and where it comes from. Make tests of private methods, not just
 //  close-ups.
 
+// fixme testing fork and merge request
+
```
Comment (xydan83) [inline test/nodesetexporter/NodesetExporterLoopTest.cpp:13]: Check conversation

Comment (xydan83) [reply test/nodesetexporter/NodesetExporterLoopTest.cpp:13]: feefe

**Comment by youngerson** [comment]
коммент

**Review by xydan83 — COMMENTED** [review]
ewfewfewf

File: test/nodesetexporter/NodesetExporterLoopTest.cpp (hunk 1 above)
Comment (youngerson) [reply test/nodesetexporter/NodesetExporterLoopTest.cpp:13]: ответ

**Review by xydan83 — APPROVED** [review]

**Pull request closed without merging**